src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -331,12 +332,29 @@ object DatabaseConversions {
       )
   }
 
+  private val convertProposals: PartialFunction[(Block, OperationHash, Operation), Tables.OperationsRow] = {
+    case (block, groupHash, Proposals(source, period, proposals)) =>
+      Tables.OperationsRow(
+        operationId = 0,
+        operationGroupHash = groupHash.value,
+        kind = "proposals",
+        blockHash = block.data.hash.value,
+        blockLevel = block.data.header.level,
+        timestamp = toSql(block.data.header.timestamp),
+        internal = false,
+        proposal = proposals.map(_.toString()),
```

Comment:
wouldn't it make more sense to call this field and the related db-column as `proposals`?

Reply:
I would also suggest using our custom-made `concatenateToString` or `toCommaSeparated` methods, instead of the default `list.toString`.
They're in scope here.

This will give us more control and visibility on the actual output format, instead of relying on the scala libraries defaults.

Reply:
proposal was already a table in the db and used by ballots as well. Does it make sense to have a "proposal" and "proposals" field? and alrighty on the concatenateToString idea